items/src/main/scala/weco/api/items/services/SierraItemUpdater.scala

```diff
@@ -0,0 +1,84 @@
+package weco.api.items.services
+
+import grizzled.slf4j.Logging
+import weco.api.stacks.models.SierraItemIdentifier
+import weco.api.stacks.services.SierraService
+import weco.catalogue.internal_model.identifiers.{IdState, IdentifierType}
+import weco.catalogue.internal_model.locations.{
+  AccessCondition,
+  PhysicalLocation
+}
+import weco.catalogue.internal_model.work.Item
+import weco.catalogue.source_model.sierra.identifiers.SierraItemNumber
+
+import scala.concurrent.{ExecutionContext, Future}
+
+/** Updates the AccessCondition of sierra items
+  *
+  *  This provides an up to date view on whether a hold
+  *  can be placed on an item.
+  *
+  *  @param sierraService
+  *  @param executionContext
```

Comment:
Do we need these @​param declarations? Are they adding anything?

Reply:
It might make it look nice if anyone ever generated scaladocs (the template is auto generated in Intellij), but it's unlikely. Will remove.